jupyterhub/templates/hub/configmap.yaml

```diff
@@ -38,6 +38,11 @@ data:
   {{- end }}
   {{- end }}
 
+  {{ if eq .Values.auth.type "custom" -}}
+  auth.custom.class-name: {{ .Values.auth.custom.className | quote }}
+  auth.custom.traitlets: {{ toJson .Values.auth.custom.traitlets | quote }}
```

Comment:
can we just call this 'auth.custom.config' rather than traitlets? Otherwise this is good to go I think!

Sorry for the nitpick!